web_timeline/static/src/js/web_timeline.js

```diff
@@ -111,7 +104,7 @@ odoo.define('web_timeline.TimelineView', function (require) {
             this.date_delay = attrs.date_delay;
             this.no_period = this.date_start == this.date_stop;
             this.zoomKey = attrs.zoomKey || '';
-            this.default_window = attrs.default_window || 'fit';
+            this.mode = attrs.mode || 'fit';
```

Comment:
The rest of the patch is OK, but I see no reason for breaking backwards compatibility by changing the option name. Can't we leave the `default_window` name as it was before?

Reply:
I added the attribute 'default_window' just before this PR. this PR intend to change the 'default_window' behavior, to a more usable one.
The new parameter is the same as the default Odoo one (same functionality), and to facilitate the use of the timeline view, it seems best to use the same attribute name for the same behavior in similar view ([see Odoo calendar view default view mode](https://github.com/odoo/odoo/blob/10.0/addons/web_calendar/static/src/js/web_calendar.js#L127))

I will change the attribute to be backwards compatible, using both syntax, like : 
```javascript
this.mode = attrs.mode || attrs.default_window || 'fit';
```
Is it okay ?

Reply:
In such case, then please break backwards compatibility 😊